web/locales/de.json

```diff
@@ -64,1078 +104,1138 @@
-          "CA-SK": {
+        },
+        "CA-SK": {
             "countryName": "Kanada",
             "zoneName": "Saskatchewan"
-          },
-          "CA-YT": {
+        },
+        "CA-YT": {
             "countryName": "Kanada",
             "zoneName": "Yukon"
-          },
-          "CC": {
+        },
+        "CC": {
             "zoneName": "Kokosinseln"
-          },
-          "CD": {
+        },
+        "CD": {
             "zoneName": "Demokratische Republik Kongo"
-          },
-          "CF": {
+        },
+        "CF": {
             "zoneName": "Zentralafrikanische Republik"
-          },
-          "CG": {
+        },
+        "CG": {
             "zoneName": "Kongo"
-          },
-          "CH": {
+        },
+        "CH": {
             "zoneName": "Schweiz"
-          },
-          "CI": {
+        },
+        "CI": {
             "zoneName": "Elfenbeinküste"
-          },
-          "CK": {
+        },
+        "CK": {
             "zoneName": "Cook Inseln"
-          },
-          "CL-SING": {
+        },
+        "CL-SING": {
             "countryName": "Chile",
             "zoneName": "SING"
-          },
-          "CL-SIC": {
+        },
+        "CL-SIC": {
             "countryName": "Chile",
             "zoneName": "SIC"
-          },
-          "CL-SEM": {
+        },
+        "CL-SEM": {
             "countryName": "Chile",
             "zoneName": "SEM"
-          },
-          "CL-SEA": {
+        },
+        "CL-SEA": {
             "countryName": "Chile",
             "zoneName": "SEA"
-          },
-          "CM": {
+        },
+        "CM": {
             "zoneName": "Kamerun"
-          },
-          "CN": {
-            "zoneName": "China"
-          },
-          "CO": {
+        },
+        "CN": {
+            "zoneName": "Volksrepublik China"
+        },
+        "CO": {
             "zoneName": "Kolumbien"
-          },
-          "CR": {
+        },
+        "CR": {
             "zoneName": "Costa Rica"
-          },
-          "CU": {
+        },
+        "CU": {
             "zoneName": "Kuba"
-          },
-          "CV": {
+        },
+        "CV": {
             "zoneName": "Kap Verde"
-          },
-          "CW": {
+        },
+        "CW": {
             "zoneName": "Curaçao"
-          },
-          "CX": {
+        },
+        "CX": {
             "zoneName": "Weihnachtsinsel"
-          },
-          "CY": {
+        },
+        "CY": {
             "zoneName": "Zypern"
-          },
-          "CZ": {
+        },
+        "CZ": {
             "zoneName": "Tschechien"
-          },
-          "DE": {
+        },
+        "DE": {
             "zoneName": "Deutschland"
-          },
-          "DJ": {
+        },
+        "DJ": {
             "zoneName": "Dschibuti"
-          },
-          "DK-DK1": {
+        },
+        "DK": {
+            "zoneName": "Dänemark"
+        },
+        "DK-DK1": {
             "countryName": "Dänemark",
             "zoneName": "West"
```

Comment:
I think this could be updated for consistency with Norway, Brazil, etc? DK2 as well

Reply:
Thanks! Fixed it :)